exe/solr_wrapper

```diff
@@ -98,6 +102,11 @@ end
 
 instance = SolrWrapper.instance(options)
 
+if instance.debug?
+  puts instance.as_json
```

Comment:
What's the use case for this data? I'm imagining it's a person who will do something with it, and does not need to be machine actionable? 

If that's the case, I have a mild preference for a) outputing to stderr, b) using a non-json format  (YAML, I guess, might make sense.. and would let us align with the on-disk configuration?), and c) not exit, but continue onto the requested command. I also wonder what the relationship between this flag and `--verbose` should be.

If there's some machine-actionable use, I'd prefer we not use `debug` as flag, and instead add named subcommand (e.g. "print-config" or similar).